packages/server/utils/parser.ts

```diff
@@ -23,17 +18,20 @@ export const parseForm = (buffer: Buffer | string, headers: ALBEventHeaders) =>
     let form = {} as ParsedFormDataOptions;
 
     bb.on('file', (fieldname: string, file: Readable, fileinfo: FileInfo) => {
-      const temp: FormData = { file: [], fieldname: '' };
+      const chunks: Buffer[] = [];
       file.on('data', (data: Buffer) => {
-        temp.file.push(data);
+        log.info(`Received ${data.length} bytes for field ${fieldname}`);
```

Comment:
A lot of logging, maybe some of this could be downgraded to log.debug, at least later on.